lib/game.rb

```diff
@@ -84,53 +90,59 @@ def display_boards
     puts @cpu_board.render
     puts "===========PLAYER BOARD==========="
     puts @player_board.render(true) #ask about writing this test
+    puts " "
   end
   
   def player_shot
     puts "Where do you want to shoot?"
-      target = gets.chomp.upcase
-      while @cpu_board.valid_coordinate?(target) == false
-        puts "Please select a target coordinate on the board"
-        target = gets.chomp.upcase
-      end
-      while @cpu_board.cells[target].fired_upon? == true
-        puts "You have aleady fired upon that coordinate"
-        gets.chomp.upcase
-      end
-      @cpu_board.cells[target].fire_upon
-      if @cpu_board.cells[target].empty?
-        puts "Miss!"
-      elsif @cpu_board.cells[target].empty? == false
-        if @cpu_board.cells[target].ship.health >= 1 
-        puts "Hit!"
-        else
-          puts "You sunk my #{@cpu_board.cells[target].ship.name}!"
-          # if @cpu_cruiser.sunk? && @cpu_submarine.sunk?
-          #   puts "Congrats! You've Won!!"
-          # end
+      @player_shot = gets.chomp.upcase
+        while @cpu_board.valid_coordinate?(@player_shot) == false
+          puts "Please select a target coordinate on the board"
+          @player_shot = gets.chomp.upcase
         end
-      end
+        while @cpu_board.cells[@player_shot].fired_upon? == true
+          puts "You have aleady fired upon that coordinate"
+          @player_shot = gets.chomp.upcase
+        end
+        @cpu_board.cells[@player_shot].fire_upon        
   end
     
   def cpu_shot
-    target = @player_board.cells.keys.sample
-    while @player_board.cells[target].fired_upon? == true
-      target = @player_board.cells.keys.sample
+    @cpu_shot = @player_board.cells.keys.sample
+    while @player_board.cells[@cpu_shot].fired_upon? == true
```

Comment:
Great to change target to an instance variable that is dynamic and now it's easier to see the difference.